org.adempiere.base/src/org/compiere/model/MUserRoles.java

```diff
@@ -64,6 +64,7 @@
 		final String whereClause = I_AD_User_Roles.COLUMNNAME_AD_User_ID+"=?";	
 		List<MUserRoles> list = new Query(ctx,I_AD_User_Roles.Table_Name,whereClause,null)
 		.setParameters(AD_User_ID)
+		.setClient_ID()

```

Comment:
This seems wrong, filtering by AD_Client_ID this way will exclude the roles the user can have in System tenant.

I think the filter must be something like adding in the whereClause
`AND AD_Client_ID IN (0,?)`
and adding the clientID to the parameters